addons/mail/controllers/main.py

```diff
@@ -275,3 +275,15 @@ def message_get_suggested_recipients(self, model, res_ids):
         except:
             return {}
         return records._message_get_suggested_recipients()
+
+    @http.route('/mail/channel_call_notify', type="json", auth="user")
```

Comment:
Those could be moved into a discuss.py file, holding controllers specific for discuss. That way we distinguish core mail from discuss, easing expected code move in a near future.

Reply:
Agree, we actually have a discuss controller in another PR, this should be integrated into it when both PR are rebased together.